src/shared/components/VideoPreviewBase/VideoPreviewBase.tsx

```diff
@@ -345,6 +349,11 @@ const VideoPreviewBase: React.FC<VideoPreviewBaseProps> = ({
                   <SvgGlyphMore />
                 </KebabMenuIconContainer>
                 <ContextMenu contextMenuOpts={contextMenuOpts}>
+                  {onOpenInTabClick && (
+                    <ContextMenuItem icon={<SvgGlyphNewTab />} onClick={onOpenInTabClick}>
```

Comment:
Thinking of it now, we will need to call this something else I think. We already use "Open in new tab" for opening video for edit (in new tab of video sheet). Let's try using `SvgGlyphPlay` instead and make text `Play in Joystream`